### PR TITLE
Shenli/fix jdbc auth

### PR DESCRIPTION
**Pull request by shenli**
Fix jdbc auth error

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: tidb-server/server/server.go [hunk 1]
```diff
@@ -63,6 +63,20 @@ func (s *Server) releaseToken(token *Token) {
 	s.concurrentLimiter.Put(token)
 }
 
+// Generate a random string using ASCII characters but avoid seperator character.
+// See: https://github.com/mysql/mysql-server/blob/5.7/mysys_ssl/crypt_genhash_impl.cc#L435
+func randomBuf(size int) []byte {
+	buf := make([]byte, size)
+	rand.Seed(time.Now().UTC().UnixNano())
```
Comment (ngaut) [inline tidb-server/server/server.go]: Call Seed once is fine. I think you may call it when in initialization function of package .

Comment (shenli) [inline tidb-server/server/server.go]: PTAL

**Comment by siddontang** [comment]
LGTM

**Pull request merged**